src/conn.rs

```diff
@@ -363,6 +386,63 @@ impl LdapConnAsync {
         Ok((conn, ldap))
     }
 
+    #[cfg(feature = "tls-native")]
+    async fn create_tls_stream(
+        settings: LdapConnSettings,
+        hostname: &str,
+        stream: TcpStream,
+    ) -> Result<TlsStream<TcpStream>> {
+        let connector = match settings.connector {
+            Some(connector) => connector,
+            None => LdapConnAsync::create_connector(&settings),
+        };
+        TokioTlsConnector::from(connector)
+            .connect(hostname, stream)
+            .await
+            .map_err(LdapError::from)
+    }
+
+    #[cfg(not(feature = "tls-native"))]
```

Comment:
This should be `cfg(feature = "tls-rustls")` to cover the case with no TLS at all.

Reply:
It's what I did before, but than the all-feature flag won't work :-/

Reply:
Don't mind this comment, I tested it and it works :)